XLS-0075d-account-permission-delegation/README.md

```diff
@@ -0,0 +1,314 @@
+<pre>
+Title:       <b>Account Permission Delegation</b>
+
+Author:      <a href="mailto:mvadari@ripple.com">Mayukha Vadari</a>
+
+Affiliation: <a href="https://ripple.com">Ripple</a>
+</pre>
+
+# Account Permission Delegation
+
+## Abstract
+
+This proposal introduces an authorization delegation mechanism to enhance the flexibility and usability of XRPL accounts.
+
+Currently, critical issuer actions, such as authorizing trustlines, require direct control by the account's keys, hindering operational efficiency and complex use cases. By empowering account holders to selectively delegate specific permissions to other accounts, this proposal aims to enhance account usability without compromising security. This mechanism will unlock new possibilities for XRPL applications, such as multi-party workflows and advanced account management strategies.
+
+## 1. Overview
+
+We propose:
+* Creating a `Delegate` ledger object.
+* Creating a `DelegateSet` transaction type.
+
+We also propose modifying the transaction common fields.
+
+This feature will require an amendment, tentatively titled `PermissionDelegation`.
+
+### 1.1. Terminology
+
+* **Delegating account**: The main account, which is delegating permissions to another account.
+* **Delegate**: The account that is having permissions delegated to it.
+
+### 1.2. Basic Flow
+
+Isaac, a token issuer, wants to set up his account to follow security best practices and separation of responsibilities. He wants some of his employees to have some sub-permissions on the account, and manage their own separate keys - Alice to manage token issuing and Bob to manage trustlines. He is also working with Kylie, a KYC provider, who he wants to be able to authorize trustlines but not have any other permissions (as she is an external party).
+
+He can authorize:
+* Alice's account for the `Payment` transaction permission.
+* Bob's account for the `TrustSet` transaction permission.
+* Kylie's account for the `TrustlineAuthorize` granular permission.
+
+The full set of available permissions is listed in [XLS-74d, Account Permissions](https://github.com/XRPLF/XRPL-Standards/discussions/217)
+
+## 2. On-Ledger Object: `Delegate`
+
+This object represents a set of permissions that an account has delegated to another account, and is modeled to be similar to [`DepositPreauth` objects](https://xrpl.org/docs/references/protocol/ledger-data/ledger-entry-types/depositpreauth).
+
+### 2.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`LedgerIndex`| ✔️|`string`|`Hash256`|The unique ID of the ledger object.|
+|`LedgerEntryType`| ✔️|`string`|`UInt16`|The ledger object's type (`Delegate`).|
+|`Account`| ✔️|`string`|`AccountID`|The account that wants to authorize another account.|
+|`Authorize`| ✔️|`string`|`AccountID`|The authorized account.|
```

Comment:
How about naming it `AuthorizedAccount` instead of `Authorize`? IMO using `verb` is a bit confusing and doesn't have a context.

Reply:
It's copied from `DepositPreauth`